app/controllers/concerns/json_web_token.rb

```diff
@@ -0,0 +1,16 @@
+require 'jwt'
+
+module JsonWebToken
+  extend ActiveSupport::Concern
+  SECRET_KEY = Rails.application.secret_key_base
+
+  def jwt_encode(payload, exp = 7.days.from_now)
+    payload[:exp] = exp.to_i
+    JWT.encode(payload, SECRET_KEY)
+  end
+
+  def jwt_decode(token)
+    decoded = JWT.decode(token, SECRET_KEY)[0]
```

Comment:
After to install everything (bundle, db, etc.) When I run the app, I'm able to see the `api-docs` but not the same for endpoint, I've this error when I try it:

<img width="1054" alt="image" src="https://user-images.githubusercontent.com/20563252/231809766-5397b1df-a390-4a60-8914-49d0559ef7d0.png">

Please, provide the secret key to install the projecy locally 🐱